src/app/AttributeCache.cpp

```diff
@@ -219,5 +274,112 @@ CHIP_ERROR AttributeCache::GetStatus(const ConcreteAttributePath & path, StatusI
     return CHIP_NO_ERROR;
 }
 
+void AttributeCache::GetSortedFilters(std::vector<std::pair<DataVersionFilter, size_t>> & aVector)
+{
+    for (auto const & endpointIter : mCache)
+    {
+        EndpointId endpointId = endpointIter.first;
+        for (auto const & clusterIter : endpointIter.second)
+        {
+            if (!clusterIter.second.mCommittedDataVersion.HasValue())
+            {
+                continue;
+            }
+            DataVersion dataVersion = clusterIter.second.mCommittedDataVersion.Value();
+            uint32_t clusterSize    = 0;
+            ClusterId clusterId     = clusterIter.first;
+
+            for (auto const & attributeIter : clusterIter.second.mAttributes)
+            {
+                if (attributeIter.second.Is<StatusIB>())
+                {
+                    clusterSize += 2; // Status + ClusterStatus = 2 bytes
```

Comment:
A StatusIB is at least 3-4 bytes when encoded as TLV, depending on how we're counting things....